tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -96,6 +99,7 @@ def setup_class(cls):
         cls.s3_mp_test_obj = S3MultipartTestLib(endpoint_url=S3_CFG["s3_url"])
         cls.test_file = "ha_mp_obj"
         cls.test_dir_path = os.path.join(TEST_DATA_FOLDER, "HATestMultipartUpload")
+        cls.remote_log_dir = "/root/log/latest"
```

Comment:
is it require in setup class? 
I guess this is only require for test : test_server_pod_restart_directed_ios?

Reply:
Yes, required for one test as of now, but it is required in teardown as well, so have to keep in setup.